presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -657,6 +665,26 @@ public TimeStat getNextPageStats()
         return handler.process(body);
     }
 
+    @VisibleForTesting
+    static Optional<String> extractAddress(String address)
+    {
+        Matcher matcher = ADDRESS_PATTERN.matcher(address);
+
+        if (!matcher.matches()) {
+            return Optional.empty();
```

Comment:
previously if the address was eg `<ip>` (no port), it would still be accepted

i understand this is conscious not to accept it (perhaps impossible), but could be good to mention in cmt msg, or even splits commits into

* validate address pattern
* add support for hostname/

Reply:
Correct. It was accepted but it never happened in practice, as ES doesn't produce that kind of value.

I'll add a comment in the commit message. I think splitting it is overkill, as I'd be just adding a commit and erase its effect on the next one.